components/suggestion/at_mention_provider/at_mention_suggestion.jsx

```diff
@@ -82,6 +84,22 @@ export default class AtMentionSuggestion extends Suggestion {
                     )}
                 </FormattedMessage>
             );
+        } else if (user.type === Constants.MENTION_GROUPS) {
+            username = user.display_name;
+            description = `- (${user.display_name})`;
+            icon = (
+                <FormattedMessage
```

Comment:
same here - renaming `user` to something more appropriate would make it less confusing i think

Reply:
agree